script.js

```diff
@@ -317,8 +317,8 @@ function filter() {
     if (curso !== "" && curso !== "Curso") {
         nodeData.curso = capitalizeFirstLetter(curso);
     }
-    if (hierarquia !== "" && hierarquia !== "Estagio") {
-        nodeData.estagio = capitalizeFirstLetter(hierarquia);
+    if (hierarquia !== "" && hierarquia !== "hierarquia") {
```

Comment:
Isn't the H capitalized?

Reply:
Well spotted this was breaking the filter. Fixed